### PR TITLE
[doc] remove the CODE_STYLE.md ref

### PR DESCRIPTION
**Pull request by Miss-you**
### What this PR does / why we need it:

Remove the CODE_STYLE.md ref which would cause invalid reference.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Once we've discussed your changes and you've got your code ready, make sure that
 ## Check code style and test case style
 
 * code style
-    * Please take a look at [OpenResty Lua Coding Style Guide](CODE_STYLE.md).
+    * Please take a look at [Lua Coding Style Guide](https://github.com/luarocks/lua-style-guide).
```
Comment (membphis) [inline Contributing.md:97]: we do not use `luarocks Lua style`, the indented is 3 spaces. 

but we are using 4 spaces.

Comment (moonming) [reply Contributing.md:97]: we can still using https://github.com/apache/incubator-apisix/blob/v1.3/CODE_STYLE.md, and remove all `openresty` in this file.

Comment (moonming) [reply Contributing.md:97]: fixed by https://github.com/apache/incubator-apisix/pull/1874

Comment (Miss-you) [reply Contributing.md:97]: ok

**Pull request closed without merging**